src/kaggle/api/kaggle_api_extended.py

```diff
@@ -4167,3 +4350,24 @@ def increment(self, length):
             length: bytes to increment the reader by
         """
         self.progress_bar.update(length)
+
+
+class FileList(object):
+    def __init__(self, init_dict):
+        self.error_message = ''
+        files = init_dict['files']
+        if files:
+            for f in files:
+                if 'size' in f:
+                    f['totalBytes'] = f['size']
+            self.files = [File(f) for f in files]
+        else:
+            self.files = {}

```

Comment:
are the types here in both conditional paths the same?

it looks like we'll end up with a list if files is present, otherwise we set a dict here. in practice idk if it matters, but we probably want this to be same type?